SimPEG/electromagnetics/static/induced_polarization/simulation.py

```diff
@@ -12,8 +13,8 @@
 from ...base import BaseEMSimulation
 
 from ..resistivity.fields import FieldsDC, Fields_CC, Fields_N
-from ..resistivity import Problem3D_CC as BaseProblem3D_CC
-from ..resistivity import Problem3D_N as BaseProblem3D_N
+from ..resistivity import Simulation3DCellCentered as BaseSimulation3DCellCentered
```

Comment:
No 🇨🇦-spelling to mess with everyone `Centred`?? 😉

Reply:
I did put Alias's for them in the `__init__.py` files already, so someone should be able to access both versions :)